administrator/language/de-DE/de-DE.plg_fields_editor.ini

```diff
@@ -4,5 +4,17 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt, see LICENSE.php
 ; Note : All ini files need to be saved as UTF-8
 
+PLG_FIELDS_EDITOR="Felder - Editor"
+PLG_FIELDS_EDITOR_LABEL="Editor (%s)"
+PLG_FIELDS_EDITOR_PARAMS_BUTTONS_HIDE_DESC="Verbirgt die Buttons in der Liste."
+PLG_FIELDS_EDITOR_PARAMS_BUTTONS_HIDE_LABEL="Buttons verbergen"
+PLG_FIELDS_EDITOR_PARAMS_HEIGHT_DESC="Bestimmt die Höhe (in Pixel) des WYSIWYG Editors - Standart ist 250px."
+PLG_FIELDS_EDITOR_PARAMS_HEIGHT_LABEL="Höhe"
+PLG_FIELDS_EDITOR_PARAMS_SHOW_BUTTONS_DESC="Die Buttons werden angezeigt."
```

Comment:
Sollen die Editor Buttons angezeigt werden?

engl.
Should the editors-xtd plugin buttons be shown?